ambition_utils/rrule/models.py

```diff
@@ -330,16 +330,20 @@ def generate_dates(self, num_dates=20):
             # Convert to time zone
             date_with_tz = fleming.attach_tz_if_none(d, self.time_zone)
             date_in_utc = fleming.convert_to_tz(date_with_tz, pytz.utc, True)
-            dates.append(date_in_utc)
-
-            for x in range(0, num_dates):
+            
+            if not start_date or date_in_utc > start_date:
```

Comment:
would this be the same if written as `if start_date and start_date < date_in_utc`?

Reply:
No. If you want to drop the leading `not` then it could be an `if/else` (with an empty `if`) or `if (start_date and start_date < date_in_utc) or not start_date:`